lib/common/windowController.py

```diff
@@ -162,7 +162,7 @@ def _get_window_rect_darwin(self):
                             if application_obj.isrunning():
                                 # move window position
                                 # Reference: https://www.macosxautomation.com/applescript/firsttutorial/11.html
-                                rect = application_obj.windows.bounds.get()
+                                rect = application_obj.windows.bounds.get()[0]
```

Comment:
How about checking the type (list) and length (>0) of `application_obj.windows.bounds.get()` before getting the values?